x-pack/plugins/security_solution/public/detection_engine/rule_creation_ui/pages/rule_creation/index.tsx

```diff
@@ -567,27 +567,38 @@ const CreateRulePageComponent: React.FC = () => {
     () => (
       <>
         <EuiHorizontalRule margin="m" />
-        <div
-          style={{
-            display: activeStep === RuleStep.aboutRule ? undefined : 'none',
-          }}
-        >
-          <StepAboutRule
-            ruleType={defineStepData.ruleType}
-            machineLearningJobId={defineStepData.machineLearningJobId}
-            index={defineStepData.index}
-            dataViewId={defineStepData.dataViewId}
-            timestampOverride={aboutStepData.timestampOverride}
-            isLoading={isCreateRuleLoading || loading}
-            form={aboutStepForm}
-          />
-
-          <NextStep
-            dataTestSubj="about-continue"
-            onClick={aboutRuleNextStep}
-            isDisabled={isCreateRuleLoading}
-          />
-        </div>
+        {
+          // TODO: https://github.com/elastic/kibana/issues/161456
+          // The About step page contains EuiRnage component which does not work properly within memoized parents.
+          // In this case we memoize the StepAboutRule and initially this component is offscreen since the Define step page is default.
+          // The About step page is not displayed due to `style.display ='none'` and this breaks EuiRange.
+          // EUI team suggested not to memoize EuiRange/EuiDualRange: https://github.com/elastic/eui/issues/6846
+          // Related ticket: https://github.com/elastic/kibana/issues/160561
+          // NOTE: We should remove this workaround once EUI team fixed EuiRange.
+          activeStep !== RuleStep.aboutRule ? null : (
```

Comment:
We need the `StepAboutRule` component to remain in the DOM even when it's not the active step, otherwise the formData gets cleared. Perhaps triggering an extra re-render at some point while the component is setting up could be a workaround for the risk score track issue?

Reply:
Ah ok, that's why probably so many tests are broken :-) Will check what we can do to re-render action step after it appeared.